apps/fbis-ccf/behaviours/submit.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+const config = require('../../../config');
+const sendEmail = require('../../../lib/utils').sendEmail;
```

Comment:
made it reusable, saweeeeet sugaaaar

![Alt Text](https://media.giphy.com/media/PQKlfexeEpnTq/giphy.gif)